doc/env.md

```diff
@@ -61,3 +61,17 @@ Napi::Error Napi::Env::GetAndClearPendingException();
 ```
 
 Returns an `Napi::Error` object representing the environment's pending exception, if any.
+
+### RunScript
+
+```cpp
+Napi::Value Napi::Env::RunScript(____ script);
+```
+- `[in] script`: A string of JavaScript code.
+
+Runs a string containing JavaScript code and returns its result.
```

Comment:
I suggest to change like this: Runs JavaScript code contained in a string and returns its result.